packages/identity/src/errors/TankerError.ts

```diff
@@ -1,8 +1,9 @@
-import { BaseError } from 'make-error';
-
-export class TankerError extends BaseError {
+export class TankerError extends Error {
   constructor(name: string = 'TankerError', message?: string) {
     super();
+
+    // Set the prototype explicitly.
+    Object.setPrototypeOf(this, TankerError.prototype);
```

Comment:
Ultimate nitpick: skip a line below?

I'm kind of seeing two distinct sections here:

```
// Inheritance
super();
Object.setPrototypeOf(this, TankerError.prototype);

// Customization (e.g. attributes)
this.name = name;
....
```

But this is not obvious with the current style.

Still a nitpick, feel free to discard.